lsp-pyright.el

```diff
@@ -174,7 +179,8 @@ Current LSP WORKSPACE should be passed in."
           (with-current-buffer it
             (lsp--spinner-start)))))
     )
-  (lsp--info "Pyright language server is analyzing..."))
+  (when lsp-pyright-show-analyzing
```

Comment:
IMHO this should be changed to lsp-log which is not displayed at all.

Reply:
Do you mean, remove the `lsp-pyright-show-analyzing` variable, and replace `lsp--info` with `lsp--log`? It sounds fine by me, and I could make that change.

Reply:
Yes. But let see what the other maintainers think about that. I think that in general emacs users prefer less noise.

Reply:
Changing `lsp--info` to `lsp-log` looks good to me too.